src/style.css

```diff
@@ -1,3 +1,75 @@
 body {
   margin: 0;
+  font-family: "Roboto", sans-serif;
+  --blue: #00acdc;
+}
+
+.header {
+  padding: 0;
+  margin: 0 auto;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+
+}
+
+.logo__img {
+  display: block;
+  width: 40px;
+  height: 40px;
+  margin-left: 50px;
```

Comment:
move the indents to the 1st place, it should be me in the header using horizontal padding

Reply:
i dont get it, what do you mean?

Reply:
add `padding: 0 50px;` to `.header`
and remove `margin-left: 50px;` from `.logo__img` and `margin-right: 50px;` from `.nav`